### PR TITLE
added support for instana agent in taupage

### PR DESCRIPTION
**Pull request by rajatparida86**
Don't merge. work under progress

### CODE REVIEW

**Comment by apfeiffer85** [comment]
👍

**Comment by rajatparida86** [comment]
👍

**Comment by apfeiffer85** [comment]
👍

**Comment by rajatparida86** [comment]
👍

**Comment by apfeiffer85** [comment]
👍

File: build/setup.d/21-install-instana-agent.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#check if INSTANA_AGENT_KEY exists, if not cancel the installation process.
+if [-z "${INSTANA_AGENT_KEY}"]; then
```
Comment (elgris) [inline build/setup.d/21-install-instana-agent.sh]: Do we want `INSTANA_AGENT_KEY` to be mandatory or optional? If it should be optional, then maybe just log the error and proceed without exit?

**Comment by christianberg** [comment]
👍

**Comment by vibhory2j** [comment]
@rajatparida86  init.sh appears to say both appdynamics machine agent and instana agent will exist together?

**Pull request merged**